pages/02.reference/01.chat/05.ChatViewerList/doc.md

```diff
@@ -0,0 +1,59 @@
+# Chat Viewer List V2 Endpoints
+
+## GET /api/v2/chats/{channelId}/users
+
+### Request
+
+#### URI Parameters
+* channelId: uint - The ID of the channel you wish to chat in.
+
+#### Query Parameters
+
+* limit: uint? [1≤integer≤100] - Amount of items per page.
+* showLurkers: boolean? - Show lurkers in the channel for staff user (Authed API and requires JWT Token)
+* continuationToken: string? - ContinuationToken to paginate through all chat users in the channel. See [Continuation Tokens](TODO) for more info on how these work
+* username: string? - Name fragment to search users by.
+
+
+### Response
+
+**Success Http Code**: 200 OK
+
+#### Headers
+
+* link:  string - [RFC5988](https://tools.ietf.org/html/rfc5988) links, which contains links next page. Example: `<some/path?limit=50&showLurkers=false&continuationToken=XXX>; rel="next"`
+
+#### Body
+
+**Media Type**: application/json
+
+**Data Type**: Array<[ChatUser](https://dev.mixer.com/rest/index.html#Role)>
```

Comment:
We may need to change definition of [ChatUser](https://dev.mixer.com/rest/index.html#Role) as this includes user information as well

Reply:
This is the wrong link good spot